README.md

```diff
@@ -1 +1,66 @@
-# library-js
+# Library-js
+
+> Libary-js is a simple introductory javascript project to demonstrate how to create a form using HTML and use Javascript to populate it.
+
```

Comment:
## Revamp the README 🌞 

- [ ] In line with keeping professional repositories, we need to ensure that our README (project description/documentation) looks more professional.  Check my comments in the same below 👇